cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -30,6 +30,7 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
     export KUBEVIRT_MEMORY_SIZE=5520M
     export KUBEVIRT_NUM_SECONDARY_NICS=2
     export KUBEVIRT_WITH_CNAO=true
+    export KUBEVIRT_WITH_MULTUS_V3=true
```

Comment:
kubevirt will use V4 right ?
if so it means that on kubevirtci we are deploying and testing only V3 which isn't 
the interesting case of kubevirt
this way a problem with V4 won't be detected before bump at kubevirt

Reply:
Posting this for visibility:

Yes, currently v4 is used, the next step we are going to take is to change kubevirt to use Multus v3 by default (by changing the relevant lanes to specify `KUBEVIRT_WITH_MULTUS_V3=true`).
This is why I also changed this test to use Multus v3.